### PR TITLE
/data: Hyperlink to the dex for everything

### PR DESCRIPTION
**Pull request by MacChaeger**
I know a lot more about html and javascript now than when I did in August, but it's still a little shaky.

### CODE REVIEW

File: chat.js [hunk 1]
```diff
@@ -1190,7 +1190,7 @@ Chat.getDataPokemonHTML = function (template, gen = 7) {
 Chat.getDataMoveHTML = function (move) {
 	if (typeof move === 'string') move = Object.assign({}, Dex.getMove(move));
 	let buf = `<ul class="utilichart"><li class="result">`;
-	buf += `<a data-entry="move|${move.name}"><span class="col movenamecol">${move.name}</span> `;
+	buf += `<a data-entry="move|${move.name}"><span class="col movenamecol"><a href="https://pokemonshowdown.com/dex/moves/${move.name}</a>${move.name}</span> `;
```
Comment (Zarel) [inline chat.js]: ...that is not where `</a>` goes.

Comment (MacChaeger) [reply chat.js]: whoops

File: chat.js [hunk 2]
```diff
@@ -1190,7 +1190,7 @@ Chat.getDataPokemonHTML = function (template, gen = 7) {
 Chat.getDataMoveHTML = function (move) {
 	if (typeof move === 'string') move = Object.assign({}, Dex.getMove(move));
 	let buf = `<ul class="utilichart"><li class="result">`;
-	buf += `<a data-entry="move|${move.name}"><span class="col movenamecol">${move.name}</span> `;
+	buf += `<a data-entry="move|${move.name}"><span class="col movenamecol"><a href="https://pokemonshowdown.com/dex/moves/${move.name}${move.name}</a></span> `;
```
Comment (Zarel) [inline chat.js]: Still wrong...

Comment (Zarel) [reply chat.js]: Here's what the link part should look like:

`<a href="https://pokemonshowdown.com/dex/moves/${move.id}">${move.name}</a>`

Comment (Zarel) [reply chat.js]: Like, links in the abstract look like this:

`<a href="URL">LINK TEXT</a>`

Comment (MacChaeger) [reply chat.js]: Oh, duh. I guess I'm still really bad at html.

**Pull request merged**